examples/trace_example.py

```diff
@@ -0,0 +1,33 @@
+# TODO: this file targets to demonstrate our tracing support, not expected to be checked in
```

Comment:
may move this as a notebook in test folder, so this won't block our PR merge

Reply:
moved